.github/workflows/ci.yml

```diff
@@ -31,9 +31,17 @@ jobs:
         uses: actions/checkout@v4
         with:
           path: custom
+          fetch-depth: 0
 
       - name: Install Python dependencies for custom project generation
-        run: python -m pip install pyyaml
+        run: |
+          python -m pip install pyyaml
+          python -m pip install requests
+
+      - name: Get changed info.json files and validate them
+        working-directory: custom
+        run: |
+          git diff origin/main --name-only | grep info\\.json | xargs python src/validate_info_json.py $1
```

Comment:
There are two possible approaches here:
- this one that only triggers on changed `info.json` files.
- validating every `info.json` regardless of those changing or not.

The latter has the advantage of informing CI when a ZIM goes missing but that may be unwanted or nor considered CI-responsibility. @kelson42 WDYT?

Reply:
@BPerlakiH as discussed, we can merge as it and adjust strategy later if @kelson42 wants to